SUMMARY.md

```diff
@@ -13,6 +13,7 @@
   * [USB](Debugging/USB.md)
 - Tutorials
   * [Communication Protocols](Tutorials/Communication_Protocols.md)
+  * [Interrupts](Tutorials/Interrupts.md)
   * [Making Your Own Module](Tutorials/Making_Your_Own_Module.md)
-  * [Pulse-width Modulation (PWM)](Tutorials/PWM.md)
+  * [PWM](Tutorials/PWM.md)
```

Comment:
Intentional? Better to keep the full name here. `book sm` probably did this automatically - consider renaming the file?